src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -175,7 +175,8 @@
                 // state of the configuration when test was made until reason found in
                 // https://github.com/checkstyle/checkstyle/issues/3730
                 "TYPECAST", "DOT", "TYPE_ARGUMENT", "ANNOTATION_DEF", "LITERAL_NEW",
-                "LITERAL_THROWS", "PARAMETER_DEF", "IMPLEMENTS_CLAUSE", "ANNOTATION_FIELD_DEF")
+                "LITERAL_THROWS", "PARAMETER_DEF", "IMPLEMENTS_CLAUSE", "ANNOTATION_FIELD_DEF",
+                "ENUM_CONSTANT_DEF", "PACKAGE_DEF")
```

Comment:
The reasons for the new tokens should be the same as https://github.com/checkstyle/checkstyle/pull/6380#discussion_r248938124

Reply:
@romani This check is in google config and we added them as defaults. Will this cause an issue like in https://github.com/checkstyle/checkstyle/issues/6381#issuecomment-455003077 ?

Reply:
yes, your are right, .... amount of items to rememeber to recheck is still high.
@pbludov , please recheck that this does not conflict with google style guide.

Reply:
The google style is very ambiguous:

> Annotations applying to a class, method or constructor appear immediately after the documentation block, and each annotation is listed on a line of its own (that is, one annotation per line). 
Exception: A single parameterless annotation may instead appear together with the first line of the signature
Annotations applying to a field also appear immediately after the documentation block, but in this case, multiple annotations (possibly parametrized) may be listed on the same line
There are no specific rules for formatting annotations on parameters, local variables, or types. 

That's all. Even the style for the interface definition is unclear.

Reply:
If there are no rules pertaining to package then the token can't be added to the required. Default is ok as long as google config doesn't use the default list, which it doesn't.

For ENUM_CONSTANT_DEF, should we treat this as a field under the rules? To me it makes sense. @romani Do you agree?

> The google style is very ambiguous:

Yep. :)

Reply:
In latest code I see new tokens in acceptable and default it is correct. Package token will not be considered.

For Google style we need to have proper config, so we can change config and all Check functionality as we need, but just need to got required functionality. If Check  is used default in  config, and we add new default token, we can change config to exclude new token.

Reply:
>For ENUM_CONSTANT_DEF, should we treat this as a field under the rules?

No, it is not a field. But we might ask confirmation at their issue tracker.

Reply:
@pbludov For this item, just add the reasons for not adding the tokens to the google config in the test.